src/Pos/Communication/Update.hs

```diff
@@ -26,10 +25,7 @@ submitVote
     -> UpdateVote
     -> m ()
 submitVote sendActions na voteUpd = do
-    void $ forConcurrently na $
-        -- TODO enqueueConversation on the set of peers and wait for it to
-        -- finish.
-        \addr -> sendVote sendActions addr voteUpd
+    sendVote (immediateConcurrentConversations sendActions na) voteUpd
```

Comment:
Same Q here and below, why does this use `immediateConcurrentConversations`? As I understand it, votes etc are classed as normal transactions as far as our policy system is concerned, so can go via normal route. Am I missing something or is this just temporary?